client/me/purchases/vat-info/index.tsx

```diff
@@ -44,11 +46,11 @@ export default function VatInfoPage() {
 			<Column type="sidebar">
 				<Card className="vat-info__sidebar-card">
 					<CardHeading tagName="h1" size={ 16 } isBold={ true } className="vat-info__sidebar-title">
-						{ translate( 'VAT Information' ) }
+						{ translate( 'Business Tax ID details' ) }
 					</CardHeading>
 					<p className="vat-info__sidebar-paragraph">
 						{ translate(
-							"We currently only provide VAT invoices to users who are properly listed in the VIES (VAT Information Exchange System) or the UK VAT databases. VAT information saved on this page will be applied to all of your account's receipts."
+							"We currently only provide Business Tax ID invoices to users who are properly listed in Business Tax ID databases. Business Tax ID information saved on this page will be applied to all of your account's receipts."
```

Comment:
@sirbrillig Thoughts on correcting this wording? The second and third usage of Business Tax ID could perhaps be simply tax ID

Reply:
🤔 I don't know. I think it would be fine either way. Whatever you think is best.